drivers/crypto/crypto_stm32.c

```diff
@@ -423,12 +423,10 @@ static int crypto_stm32_query_caps(const struct device *dev)
 
 static int crypto_stm32_init(const struct device *dev)
 {
-	const struct device *clk = device_get_binding(STM32_CLOCK_CONTROL_NAME);
+	const struct device *clk = DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE);
 	struct crypto_stm32_data *data = CRYPTO_STM32_DATA(dev);
 	const struct crypto_stm32_config *cfg = CRYPTO_STM32_CFG(dev);
 
-	__ASSERT_NO_MSG(clk);
-
```

Comment:
Same comment here: we need to make clear that people using `DEVICE_DT_GET()` must still validate the device before it's used.  Assert or error if that isn't true.

Reply:
So, it means that commit message isn't correct?
> A few places had __ASSERT_NO_MSG(device_get_binding()) these have been
removed since we'll get a build errof if DEVICE_DT_GET cant be
satisified,

Reply:
Same comment as where?

Reply:
Added check here for return of clock API.

Reply:
I updated the commit message